### ISSUE TITLE
Carousel Title color is hard to read

### ISSUE THREAD

**Issue by supernovia** [Automattic/newspack-blocks#548]
**Describe the bug**
Carousel inherits title link color, but always displays a dark background. If the link color is dark, it makes things hard to see.
I am noticing this in two different themes so far. It seems to depend on the link color, and whether the title links have an explicit color set.

From Publisher Two:
![Screen Shot 2020-07-20 at 5 10 27 PM](https://user-images.githubusercontent.com/3476361/87995333-6e2f4a00-caac-11ea-8812-f6b89bc03a3b.png)

From Stratford:
![Screen Shot 2020-07-20 at 5 18 32 PM](https://user-images.githubusercontent.com/3476361/87995656-586e5480-caad-11ea-9313-c98c055e6886.png)

**To Reproduce**
1. Add a Post Carousel block
2. Choose a theme with dark title colors

**Screenshots**
See above

**Expected behavior**
Have sufficient contrast between background and text, or allow users to customize block colors, re: https://github.com/Automattic/newspack-blocks/issues/382

**Comment by supernovia** [comment]
Spotted here: https://wordpress.com/forums/topic/font-color-post-carousel/

**Comment by supernovia** [comment]
@iamtakashi do you know if there's someone we can ping about this? I'm not sure why we're getting dark text on a dark background. Asking only because I saw you were pinged in the other post, and because I know you're pretty awesome 🥇

**Comment by iamtakashi** [comment]
@supernovia Thanks for the ping! I had a quick look and was able to replicate the issue. I think the CSS for the block needs to be improved to avoid the issue. 

I think that @thomasguillot is on a leave now, maybe @laurelfulford could help?

**Comment by laurelfulford** [comment]
Thanks for the ping! I made some tweaks in #553 that seem to fix for Stratford & IP2 -- there's a chance we'll need to adjust again if there's a theme out there with super specific styles that override even this fix, but we'll cross that bridge when we get to it :)

**Comment by supernovia** [comment]
Thank you!

**Comment by adekbadek** [comment]
Changes in this repository are synchronised to wordpress.com pretty infrequently – by now they should have been synchronised and the issue resolved. If this is not the case, please reopen.
